File: components_page/components/button_group/size.R
library(dashBootstrapComponents)
library(dashHtmlComponents)


button_groups <- htmlDiv(
  list(
    dbcButtonGroup(
      list(
        dbcButton("Left"), dbcButton("Middle"),
        dbcButton("Right")
      ),
      size = "lg",
      className = "mr-1",
    ),
    dbcButtonGroup(
      list(
        dbcButton("Left"), dbcButton("Middle"),
        dbcButton("Right")
      ),
      size = "md",
      className = "mr-1",
    ),
    dbcButtonGroup(
      list(
        dbcButton("Left"), dbcButton("Middle"),
        dbcButton("Right")
      ),
      size = "sm",
    )
  )
)

File: components_page/components/button_group/size.jl
using DashBootstrapComponents, DashHtmlComponents

button_groups = html_div([
    dbc_buttongroup(
        [dbc_button("Left"), dbc_button("Middle"), dbc_button("Right")],
        size="lg",
        class_name="me-1",
    ),
    dbc_buttongroup(
        [dbc_button("Left"), dbc_button("Middle"), dbc_button("Right")],
        size="md",
        class_name="me-1",
    ),
    dbc_buttongroup(
        [dbc_button("Left"), dbc_button("Middle"), dbc_button("Right")],
        size="sm",
    ),
]);
